src/qt/forms/askmultisigdialog.ui

```diff
@@ -0,0 +1,327 @@
+      <property name="geometry">
+       <rect>
+        <x>0</x>
+        <y>0</y>
+        <width>546</width>
+        <height>326</height>
+       </rect>
+      </property>
+      <attribute name="label">
+       <string>Create standard Escrow multisig</string>
+      </attribute>
+      <layout class="QGridLayout" name="escrowGridLayout">
+       <item row="4" column="2">
+        <widget class="QLineEdit" name="counterpartyAddressLineEdit"/>
+       </item>
+       <item row="5" column="2">
+        <widget class="QLineEdit" name="escrowAddressLineEdit"/>
+       </item>
+       <item row="1" column="2">
+        <widget class="QLineEdit" name="yourAddressLineEdit">
+         <property name="enabled">
+          <bool>false</bool>
+         </property>
+         <property name="font">
+          <font>
+           <italic>true</italic>
+          </font>
+         </property>
+         <property name="text">
+          <string>(will be generated automatically)</string>
+         </property>
+         <property name="readOnly">
+          <bool>true</bool>
+         </property>
+        </widget>
+       </item>
+       <item row="1" column="1">
+        <widget class="QLabel" name="yourAddressLabel">
+         <property name="text">
+          <string>Your address pubkey</string>
+         </property>
+        </widget>
+       </item>
+       <item row="5" column="1">
+        <widget class="QLabel" name="escrowLabel">
+         <property name="text">
+          <string>Escrow pubkey &lt;i&gt;(optional)&lt;/i&gt;&amp;nbsp;</string>
+         </property>
+        </widget>
+       </item>
+       <item row="4" column="1">
+        <widget class="QLabel" name="counterpartyAddressLabel">
+         <property name="text">
+          <string>Counterparty pubkey</string>
+         </property>
+        </widget>
+       </item>
+       <item row="6" column="1" colspan="2">
+        <widget class="QLabel" name="escrowExplainedLabel">
+         <property name="styleSheet">
+          <string notr="true">background-color: #f0fbf4</string>
+         </property>
+         <property name="frameShape">
+          <enum>QFrame::Panel</enum>
+         </property>
+         <property name="frameShadow">
+          <enum>QFrame::Sunken</enum>
+         </property>
+         <property name="text">
+          <string>&lt;p&gt;&lt;i&gt;Your address pubkey&lt;/i&gt; is generated automatically.
+You only need to provide a address pubkey generated by a person you are trading with (&lt;i&gt;Counterparty pubkey&lt;/i&gt;) and optionally third party (the escrow) one.&lt;/p&gt;
+&lt;p&gt;If no escrow pubkey is provided a 2/2 multisig would be generated
+(i.e. both you and the counterparty need to agree to redeem). If escrow is provided, 2/3 multisig would be generated (i.e. two of you three need to agree).&lt;/p&gt;
+&lt;p&gt;To generate raw pubkey click button below. To import multisig address generated by a counterparty, switch to last tab.&lt;/p&gt;</string>
```

Comment:
Noticed this from watching the screen, the text is a little confusing.

Perhaps better as:

Your **public key** is generated automatically.

You only need to provide a **public key** generated by the person you are **interacting** with (counterparty) and optionally **a third party key that acts as an escrow**.

If no escrow **key** is provided, a **2-of-2 multisig address** will be generated, **meaning that** both you and the counterparty need to **sign** to **withdraw funds**. If an escrow **key** is provided, a **2-of-3 multisig address** will be generated, **meaning that any 2 parties can sign to withdraw funds**.

To generate a **new public key**, click **the** button below. To import **a** multisig address generated by a counterparty, switch to **the** last tab.